### DISCUSSION TITLE
是否可以通过取钱和出售商业债券性质的资产来帮助抗议

### DISCUSSION THREAD

**Discussion by ALight10** [WeNeedHome/SummaryOfLoanSuspension#958] [General]
rt
对于停贷抗议,我们非直接停贷的人
是否可以通过取钱和出售商业债券性质的资产来帮助抗议

其一
我所知道现在银行必须要有存款保证金才能放贷
如果我取出自己存入在银行的一切资产
是不是等于银行的保证金更小了
银行能放贷的空间就更小了
等于银行压力会更大

其二
债券性质的资产,不指国债,主要是商业等
这部分等于我们出钱借给停贷的人
然后金融机构从中抽成
如果我们选择退场
是不是增加相关利益人士的压力
相关利益人士的资金少了
他们的压力也会更大

其三
从个人利益角度来说
我现在信不过一切金融工具
包括银行存款

其四
这种抗议没有风险
且可以通过例如河南银行事件
或者描述停贷事件的利弊
劝说周围的人一起参与
我不想成为银行等金融机构帮凶

初次对社会事件发声,如果有逻辑错误等还请多指评,谢谢

**Comment by imhappyok** [comment]
恶意取款是要被拘留的

**Comment by caesarding07** [comment]
你这是要我们去挤兑

**Comment by ydf** [comment]
最好的方式还是把钱换成外汇， 比如日元（最近日元非常便宜）

**Comment by volans19** [comment]
现在取钱都属于恶意取钱了，是违法违规的，孩子
倒是可以把银行暴雷和这个项目合过来
